external/fv3fit/fv3fit/reservoir/model.py

```diff
@@ -69,7 +69,12 @@ def load(cls, path: str) -> "ReservoirComputingModel":
 
         f = io.BytesIO(mapper[cls._READOUT_NAME])
         readout_components = joblib.load(f)
-        readout = ReservoirComputingReadout(**readout_components)
+        readout_hyperparameters = dacite.from_dict(
+            ReadoutHyperparameters, readout_components.pop("hyperparameters")
+        )
+        readout = ReservoirComputingReadout(
```

Comment:
Can you move the low-level logic on how to load a ReservoirComputingReadout into a `ReservoirComputingReadout.load` class method? Similarly for `Reservoir`. This will avoid e.g. your duplicated change in `HybridReservoirComputingModel` when these classes are modified.